text/utilities/0003-git-ssh-token.md

```diff
@@ -0,0 +1,41 @@
+# SSH Token Configuration in the Git Buildpack
+
+## Summary
+
+Add the ability to supply an SSH token to the Git buildpack that will be used
+by `git` in order to allow later buildpacks to perform operations that require
+`git` authentication.
+
+## Motivation
+
+There are some operations, such as obtaining private language modules, that
+require `git` authentication during the build process. Currently there is now
+way of authenticating as a user to enable these workflows. Adding this
+functionality to the Git buildpack would open up a wider range of workflows
+than adding the functionality on an individual buildpack basis and make the
+ability to add SSH authentication to the project universally. 
+
+## Implementation
+
+For security purposes, the token cannot be provided through an environment
+variable like most of our other buildpack configuration as this poses a risk of
+exposure by view the build process in some form of CI. Because of this
+restriction, an approach similar to the CA Certificates Buildpack will be used
+where the token will be provided in a mounted volume through a service binding.
+The `type` for the service binding will be `git-ssh-token`.
```

Comment:
I believe these are generally git tokens or passwords. Using the term ssh token might be a bit misleading since at the end, the protocol that is used is https.